lib/routes/user.js

```diff
@@ -0,0 +1,57 @@
+import express from 'express';
+import UserService from '../services/user.service';
+import UserValidator from '../validators/user.validator';
+
+const router = express.Router();
+const userService = new UserService();
+const userValidator = new UserValidator();
+
+router.param('user_id', (req, res, next, user_id) => {
+    req.user = userService.find(user_id);
```

Comment:
It will be better to move logic of endpoints into controller (create new folder controllers). Router just should describe endpoints and what functions to call to process this endpoints.

Reply:
Done